app/controllers/career_site/positions_controller.rb

```diff
@@ -166,4 +169,17 @@ def set_cors_headers
     response.headers["Access-Control-Expose-Headers"] = "X-CSRF-Token"
     response.headers["Content-Security-Policy"] = "frame-ancestors *"
   end
+
+  def set_locale
+    locale = params[:locale]
+    if locale && I18n.available_locales.include?(locale.to_sym)
+      I18n.locale = locale
+      @locale = locale
+    elsif locale.present?
+      I18n.locale = I18n.default_locale
+      flash[:alert] = I18n.t("errors.locale_not_available", language: locale)
```

Comment:
What would be if locale is a random string?

Reply:
Maybe i didn't quite understand your question, but now we are showing the message (Regardless of whether such a language exists):
![Screenshot from 2024-12-06 01-38-35](https://github.com/user-attachments/assets/5c8e026d-aa2a-4345-970c-8039c237e5be)